modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -176,16 +179,22 @@ const Modal = ({
 		<>
 			{visible && (
 				<ClayModal
+					aria-labelledby={modalLabelId}
+					center={center}
 					className="liferay-modal"
 					containerProps={{...containerProps}}
 					disableAutoClose={disableAutoClose}
 					id={id}
 					observer={observer}
+					role="dialog"
```

Comment:
I added this  `role="dialog"` for a more descriptive screen reader message, without this people will never know that this is an overlay/modal/dialog